conda/build.sh

```diff
@@ -13,8 +13,10 @@ set -x -e
 
 # C extensions must be installed with sequential BLAS library
 # https://pyscf.org/install.html#using-optimized-blas
-export CMAKE_CONFIGURE_ARGS="-DWITH_F12=OFF -DBLA_VENDOR=Intel10_64lp_seq"
+# Finding sequential BLAS is is now handled in pyscf/lib/CMakeLists.txt
+
+export CMAKE_ARGS="-DWITH_F12=OFF"
```

Comment:
Please do not change the existing compiling configurations. On conda,  mkl is configured as default dependency. The setting for BLA_VENDOR is necessary to ensure to link to correct mkl library.

Reply:
It probably would have been better to stick to the old rule "if it ain't broke, don't fix it." But in my defense, I did make sure the behavior would stay the same and it would link MKL:
https://github.com/chillenb/pyscf/blob/bc6dab5ed7ed0c2d9bd909603a371f52e14d42e3/pyscf/lib/CMakeLists.txt#L88